src/modules/Profile.rb

```diff
@@ -663,63 +665,63 @@ def Compat(__current)
     # @param file [String] path to file
     # @return [Boolean]
     def ReadXML(file)
-      tmp = Convert.to_string(SCR.Read(path(".target.string"), file))
-      l = Builtins.splitstring(tmp, "\n")
-      if !tmp.nil? && Ops.get(l, 0, "") == "-----BEGIN PGP MESSAGE-----"
-        out = {}
-        while Ops.get_string(out, "stdout", "") == ""
-          UI.OpenDialog(
-            VBox(
-              Label(
-                _("Encrypted AutoYaST profile. Enter the correct password.")
-              ),
-              Password(Id(:password), ""),
-              PushButton(Id(:ok), Label.OKButton)
+      begin
```

Comment:
np: I think you can omit the `begin` word here if you want it to cover the full method.

Reply:
well, I want to emphasize here that it is without Import, but it is right, that import would not happen if exception happen.